mppm_versioned_docs/version-1.0.0/install.md

```diff
@@ -0,0 +1,21 @@
+---
+id: install
+title: Install Multiplayer Play Mode
+description: How to install Multiplayer Play Mode
+---
+
+To begin testing your multiplayer functionality, you must first install the Multiplayer Play Mode package.
+
+1. Open Unity Hub.
+2. Select a Project.
+3. In the Editor's menu bar, navigate to **Window** > **Package Manager**.
+4. Click the plus sign ![plus sign](/img/add.png) in the Package Manager status bar.
+5. Select **Install package by name...**.
```

Comment:
This is not relevant anymore as MPPM is part of the manifest.

Reply:
@Vic-Cooper , I approved the PR already in order to not block you but this would be nice to update this section if not done yet :-)